gcp_variant_transforms/transforms/sample_info_to_bigquery.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, Union  # pylint: disable=unused-import
+
+import apache_beam as beam
+
+from gcp_variant_transforms.beam_io import vcf_header_io  # pylint: disable=unused-import
+from gcp_variant_transforms.libs import sample_info_table_schema_generator
+from gcp_variant_transforms.libs import hashing_util
+
+
+class ConvertSampleInfoToRow(beam.DoFn):
+  """Extracts sample info from `VcfHeader` and converts it to a BigQuery row."""
+
+  def process(self, vcf_header):
+    # type: (vcf_header_io.VcfHeader) -> Dict[str, Union[int, str]]
+    for sample in vcf_header.samples:
+      sample_id = hashing_util.generate_unsigned_hash_code(
+          [vcf_header.file_path, sample], max_hash_value=pow(2, 63))
```

Comment:
Let's make the max hash value `pow(2, 63) - 1` to make sure we do not reach the max INT64 value. Also, probably we need to define this value as a const in `sample_info_table_schema_generator`.

Reply:
Since we used % in generate_unsigned_hash_code, I don't think we will reach the max value, right?